### DISCUSSION TITLE
zigbee2mqtt unable to start

### DISCUSSION THREAD

**Discussion by Streamnet-coder** [Koenkk/zigbee2mqtt#20297] [General]
Hello Forum. I am new to IoT setup, finding it difficult to complete the configuration. please any help will be appreciate, please find attached my log :

[15:57:59] INFO: Preparing to start...
[15:57:59] INFO: Socat not enabled
[15:58:01] INFO: Starting Zigbee2MQTT...
Zigbee2MQTT:info  2023-12-19 15:58:07: Logging to console and directory: '/config/zigbee2mqtt/log/2023-12-19.15-58-07' filename: log.txt
Zigbee2MQTT:info  2023-12-19 15:58:07: Starting Zigbee2MQTT version 1.34.0 (commit #unknown)
Zigbee2MQTT:info  2023-12-19 15:58:07: Starting zigbee-herdsman (0.25.0)
Zigbee2MQTT:error 2023-12-19 15:58:28: Error while starting zigbee-herdsman
Zigbee2MQTT:error 2023-12-19 15:58:28: Failed to start zigbee
Zigbee2MQTT:error 2023-12-19 15:58:28: Check https://www.zigbee2mqtt.io/guide/installation/20_zigbee2mqtt-fails-to-start.html for possible solutions
Zigbee2MQTT:error 2023-12-19 15:58:28: Exiting...
Zigbee2MQTT:error 2023-12-19 15:58:28: Error: Failed to connect to the adapter (Error: SRSP - SYS - ping after 6000ms)
    at ZStackAdapter.start (/app/node_modules/zigbee-herdsman/src/adapter/z-stack/adapter/zStackAdapter.ts:103:27)
    at Controller.start (/app/node_modules/zigbee-herdsman/src/controller/controller.ts:132:29)
    at Zigbee.start (/app/lib/zigbee.ts:60:27)
    at Controller.start (/app/lib/controller.ts:101:27)
    at start (/app/index.js:107:5)

**Comment by lux73** [comment]
z2m is unable to connect to your Coordinator - which USB Stick do you have? 

Docker or Plain Installation?

**Comment by treemonkey83** [comment]
I've been having all sorts of issues with zigbee2mqtt not starting, even if i get it to start it will revert back to a error state if i restart the addon or home assistant. i have found if i stop the z2m addon and start the Silicon Labs Flasher addon, let it finish its start up (takes a couple of minutes) then stop the silicon addon. Z2M will then start! I think the reason z2m is not starting is something to do with the zigbee adaptor and by starting and stoping the silicon labs flasher addon it resets the adaptor and allows z2m to connect to it.

I am using the skyconnect dongle if it helps.
